packages/wrangler/src/deployments.ts

```diff
@@ -80,14 +84,29 @@ Source: ${sourceStr(versions.metadata.source)}\n`
 function sourceStr(source: string): string {
 	switch (source) {
 		case "api":
-			return "📡 API";
+			return "API 📡";
 		case "dash":
-			return "🖥️ Dashboard";
+			return "Dashboard 🖥️";
 		case "wrangler":
-			return "🤠 Wrangler";
+			return "Wrangler 🤠";
 		case "terraform":
-			return "🏗️ Terraform";
+			return "Terraform 🏗️";
 		default:
 			return "Other";
 	}
 }
+
+function triggerStr(trigger: string): string {
```

Comment:
I wasn't immediately clear on what this function does; maybe from a readability perspective something like `formatTrigger` would be better?